en/micro-integrator/docs/references/connectors/fhir-connector/fhir-connector-example.md

```diff
@@ -0,0 +1,318 @@
+               <fhir.readSpecificResourceById>
+                   <type>{$ctx:type}</type>
+                   <id>{$ctx:id}</id>
+                   <format>{$ctx:format}</format>
+                   <summary>{$ctx:summary}</summary>
+               </fhir.readSpecificResourceById>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+       <resource methods="POST" url-mapping="/delete">
+           <inSequence>
+               <property expression="json-eval($.base)" name="base" scope="default" type="STRING"/>
+               <property expression="json-eval($.resourceType)" name="type" scope="default" type="STRING"/>
+               <property expression="json-eval($.format)" name="format" scope="default" type="STRING"/>
+               <property expression="json-eval($.idToDelete)" name="idToDelete" scope="default" type="STRING"/>
+               <fhir.init>
+                   <base>{$ctx:base}</base>
+               </fhir.init>
+               <fhir.delete>
+                   <type>{$ctx:type}</type>
+                   <idToDelete>{$ctx:idToDelete}</idToDelete>
+               </fhir.delete>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+       <resource methods="POST" url-mapping="/update">
+           <inSequence>
+               <property expression="json-eval($.base)" name="base" scope="default" type="STRING"/>
+               <property expression="json-eval($.resourceType)" name="type" scope="default" type="STRING"/>
+               <property expression="json-eval($.format)" name="format" scope="default" type="STRING"/>
+               <property expression="json-eval($.idToUpdate)" name="idToDelete" scope="default" type="STRING"/>
+               <fhir.init>
+                   <base>{$ctx:base}</base>
+               </fhir.init>
+               <fhir.update>
+                   <type>{$ctx:type}</type>
+                   <idToUpdate>{$ctx:idToUpdate}</idToUpdate>
+                   <format>{$ctx:format}</format>
+               </fhir.update>
+               <log level="full" separator=","/>
+               <send/>
+           </inSequence>
+           <outSequence/>
+           <faultSequence/>
+       </resource>
+   </api>
+
+    ```
+3. Now we can export the imported connector and the API into a single CAR application. CAR application is the one we are going to deploy to server runtime.
+
+{! /references/connectors/exporting-artifacts.md !}
+
+## Deployment
+
+Follow these steps to deploy the exported CApp in the Enterprise Integrator Runtime. 
+
+{!references/connectors/deploy-capp.md!}
+    
+## Testing
+
+Invoke the API as shown below using the curl command. Curl Application can be downloaded from [here](https://curl.haxx.se/download.html).
+
+1. Create resource operation.
```

Comment:
Let's say wha we are going to do here.  Example : 
1. Add patient information

Reply:
Added titles as mentioned above.